tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES6",
```

Comment:
@pettinarip your question here and my response was removed due to a minor change I did.

So as to if this effects browser support, ES6 is supported by roughly 98% from [caniuse.com](https://caniuse.com/es6).

[The TypeScript docs of course also recommends ES6](https://www.typescriptlang.org/tsconfig#target)

Because there are dynamic keys in some component themes (i.e. `[$border.variable]`), the CLI will fail with the `ES5` target because dynamic keys are part of the`ES6` spec.